src/WalletLink/index.ts

```diff
@@ -0,0 +1,133 @@
+import { Buffer } from 'buffer'
+import queryString from 'query-string'
+
+export type LinkWalletRequest = {
+  requestAppId: string
+}
+
+export type LinkWalletResponse = {
+  status: 'success' | 'user_cancelled'
+  token?: string
+}
+
+export type SignHotspotRequest = {
+  universalLink: string
+  requestAppId: string
+  token: string
+  addGatewayTxn?: string
+  assertLocationTxn?: string
+}
+
+export type SignHotspotResponse = {
+  status: 'success' | 'token_not_found' | 'user_cancelled' | 'gateway_not_found'
+  assertTxn?: string
+  gatewayTxn?: string
+}
+
+export type MakerApp = {
+  universalLink: string
+  name: string
+  androidPackage: string
+  iosBundleId: string
+}
+
+const heliumStarter: MakerApp = {
+  universalLink: 'makerappscheme://',
```

Comment:
is this intended to be changeable somehow?

Reply:
My thought was that I want us to be in control of where we call back to with the signed tx. I guess makers would have to request that we add them and we'd identify them by their bundle id.